cli/src/services/session.service.ts

```diff
@@ -19,8 +19,9 @@ export class SessionService {
   readonly authPath!: string;
 
   constructor(configDirectory: string) {
-    this.configDirectory = configDirectory;
-    this.authPath = path.join(configDirectory, '/auth.yml');
+    const environmentDirectory = process.env.IMMICH_CONFIG_DIR;
```

Comment:
Just curious the reason to add this as it doesn't look like it's being used elsewhere in this PR

Reply:
yes, I was actually just saw this, too. Will push new commit in a minute

Reply:
This was an unneeded duplication, thanks for spotting. Removed.